setup.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python
```

Comment:
lol. i had one job. i've been using plain python since 2006, it'll be hard to relearn!